tls/s2n_tls13_handshake.c

```diff
@@ -85,6 +84,70 @@ int s2n_tls13_compute_shared_secret(struct s2n_connection *conn, struct s2n_blob
     return 0;
 }
 
+int s2n_tls13_compute_pq_hybrid_shared_secret(struct s2n_connection *conn, struct s2n_blob *shared_secret) {
```

Comment:
I'm out of the 1.3 loop: does this need a comment or readme explaining that is only works for exactly one ecc group and 1 kem param?

Reply:
I can certainly put such a comment there.